elliottlib/cli/__main__.py

```diff
@@ -113,18 +117,17 @@ def remove_bugs(runtime, advisory, default_advisory_type, id):
 
 
 """
+    if auto and len(id) > 0:
+        raise click.BadParameter("Combining the automatic and manual bug modification options is not supported")
+    if not auto and len(id) == 0:
```

Comment:
i prefer the [pep 8 style of checking for empty](https://www.quora.com/What-is-the-Pythonic-way-to-check-for-an-empty-set):
```suggestion
    if auto and id:
        raise click.BadParameter("Combining the automatic and manual bug modification options is not supported")
    if not auto and not id:
```